modules/division.js

```diff
@@ -47,6 +47,22 @@ module.exports = {
         const ts = moment().format('YYMMDD')
         return `https://publications.parliament.uk/pa/cm201719/cmagenda/ob${ts}.htm`
       }
+    },
+    future: {
+      help: 'generates link to Commons future business',
+      aliases: [ 'futurebusiness' ],
+      command: () => {
+        const ts = moment().format('YYMMDD')
+        return `https://publications.parliament.uk/pa/cm201719/cmagenda/fb${ts}.htm`
+      }
+    },
+    lbusiness: {
```

Comment:
`business [commons|lords]` default to commons?

Reply:
Probably a good idea but we don't support optional arguments nicely at the moment, so we'd have to do an `if (msg.args && msg.args[0] && msg.args[0].toLowerCase() === 'lords')

where's the U+XXXX PERSON ADDING TICKET TO BACKLOG emoji